src/board.py

```diff
@@ -1,12 +1,49 @@
+import string
+
 def construct_board():
-    return [[0] * 6] * 6
+    return [[0]*6 for _ in range(6)]
+
+def is_board_full(board):
+    for x, line in enumerate(board, 0):
+        for y, value in enumerate(line, 0):
+            if value == 0:
+                return False
+    return True
+
+def get_position_if_valid(board, position):  
+    try:
+        if len(position) != 2:
+            return None
+
+        x = int(position[1]) - 1
+
+        # See https://stackoverflow.com/questions/5927149/get-character-position-in-alphabet
+        y = ord(position[0].lower()) - 97
```

Comment:
that's cryptic, and I don't want to go to StackOveflow to understand. Would you mind explaining here?